src/Mealz/MealBundle/Resources/views/Meal/partials/meal.html.twig

```diff
@@ -1,5 +1,24 @@
 {% set dish = meal.dish %}
-<div class="meal-row" {{ dish.isCombinedDish ? 'data-combined="1"' }} data-slug="{{ dish.slug }}">
+{% set combinedDish = '0' %}
+{% set combinedDishIDs = '' %}
+{% set combinedDishTitle = '' %}
+{% set dishDescription = dish.description|default('content.participation.dish.no_description'|trans({},'general')) %}
+
+{% if dish.isCombinedDish %}
+    {% set combinedDish = '1' %}
+    {% set bookedDishCombination = participationService.getBookedDishCombination(app.user.profile, meal) %}
+    {% if bookedDishCombination|length > 0 %}
+        {% set combinedDishTitle = bookedDishCombination.toArray|map(d => d.title)|join(',') %}
```

Comment:
I think here you need the space, because the titles are displayed.

Reply:
Fixed